cardano-cli/test/cardano-cli-golden/files/golden/help/allegra_transaction_calculate-min-fee.cli

```diff
@@ -21,6 +25,11 @@ Available options:
   --reference-script-size NATURAL
                            Total size in bytes of transaction reference scripts
                            (default is 0).
+  --output-json            Format calculate-min-fee query output to JSON.
```

Comment:
`calculate-min-fee` appears here, because the code for this flag is shared with queries.

If we want to remove the terms `calculate-min-fee query` here, we need to generalize said parsers a bit.

Reply:
You mean `query` right?

Reply:
@carbolymer> No I meant `calculate-min-fee query` (`Format output to JSON` would still be a fine message)